modules/Bio/EnsEMBL/Variation/Pipeline/SpliceAI/SpliceAI_conf.pm

```diff
@@ -0,0 +1,195 @@
+ # All Hive databases configuration files should inherit from HiveGeneric, directly or indirectly
+use base ('Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf');
+
+sub default_options {
+    my ($self) = @_;
+
+    # The hash returned from this function is used to configure the
+    # pipeline, you can supply any of these options on the command
+    # line to override these default values.
+    
+    # You shouldn't need to edit anything in this file other than
+    # these values, if you find you do need to then we should probably
+    # make it an option here, contact the variation team to discuss
+    # this - patches are welcome!
+
+    return {
+        %{ $self->SUPER::default_options()
+        },    # inherit other stuff from the base class
+
+        hive_auto_rebalance_semaphores => 1,
+        hive_default_max_retry_count => 0,
+        hive_force_init      => 1,
+        hive_use_param_stack => 1,
+
+        pipeline_name           => $self->o('pipeline_name'),
+        main_dir                => $self->o('main_dir'), # main_dir = '/hps/nobackup2/production/ensembl/dlemos/tmp_spliceai_pipeline'
+        tmp_split_vcf_dir       => $self->o('main_dir') . '/tmp_split_vcf', # contains vcf input splited without headers
+        tmp_split_vcf_input_dir => $self->o('main_dir') . '/tmp_split_vcf_input', # contains vcf input splited with headers, these are the files used to run SpliceAI
+        tmp_output_dir          => $self->o('main_dir') . '/tmp_output', # temporary output files, still splited
+        output_dir              => $self->o('main_dir') . '/output', # final output files already merged by chromosome
+        fasta_file              => $self->o('fasta_file'), # '/hps/nobackup2/production/ensembl/dlemos/files/Homo_sapiens.GRCh38.dna.toplevel.fa'
+        gene_annotation         => $self->o('gene_annotation'), # '/homes/dlemos/work/tools/SpliceAI_files_output/gene_annotation/ensembl_gene/grch38_MANE_8_7.txt'
+        step_size               => $self->o('step_size'), # number of variants used to split the main vcf files
+        check_transcripts       => $self->o('check_transcripts'), # checks which are the new MANE transcripts for the last months, runs SpliceAI only for these ones
+        registry                => $self->o('registry'), # database where new MANE transcripts are going to be checked
+        # transcripts_file        => $self->o('transcripts_file'), # Instead of checking the new transcripts from the database, it can run only for transcripts from this file
+        output_file_name        => 'spliceai_scores_',
+
+        pipeline_wide_analysis_capacity => 100,
+
+        pipeline_db => {
+            -host   => $self->o('hive_db_host'),
+            -port   => $self->o('hive_db_port'),
+            -user   => $self->o('hive_db_user'),
+            -pass   => $self->o('hive_db_password'),
+            -dbname => $ENV{'USER'} . '_hive_' . $self->o('pipeline_name'),
```

Comment:
We need to standardise the naming of ehive pipelines. user_ehive_<pipeline_type>_<e!version>_<assembly>_<species>